### ISSUE TITLE
Display code for speciffic version of variadic.js

### ISSUE THREAD

**Issue by mradenovic** [variadicjs/variadicjs.github.io#24]
Currently, the code in the `master` branch of variadic.js is displayed on the FuncCard. That is not in sync with the version used to run the function.

**Comment by jhowardjr** [comment]
It looks like you are fetching the develop branch:

```
fetch(`https://raw.githubusercontent.com/variadicjs/variadic.js/**develop**/lib/${func}.js`)
```
Probably should be the master branch, because that is where I'm deploying the package from.

**Comment by mradenovic** [comment]
My bad, it is `develop`. It should be the commit for the version in package.json. I think it's even possible to make a list of releases, and give a user possibility to test specific release.

**Comment by mradenovic** [comment]
Too ambitious. For now it should be related to the version in package.json. If develop branch moves forward, demo code stays on the same version.

**Comment by mradenovic** [comment]
Wrong reference #24 instead of #22 addressed in #25.

**Comment by jhowardjr** [comment]
I just pushed to the develop branch by mistake, but the update should fecth the source code for the released version
